src/components/modalPrompt/tests/ModalPrompt.spec.tsx

```diff
@@ -0,0 +1,169 @@
+import { shallow, mount, ReactWrapper } from 'enzyme';
+// tslint:disable-next-line:no-unused-variable
+import * as React from 'react';
+import { ModalPrompt, IModalPromptProps } from '../ModalPrompt';
+
+describe('ModalPrompt', () => {
+  const id: string = 'modalPrompt';
+  const title: string = 'Title';
+  const confirmLabel: string = 'Confirm';
+  const cancelLabel: string = 'Cancel';
+  const message: string = 'message';
+
+  describe('<ModalPrompt />', () => {
+    it('should render without errors', () => {
+      expect(() => {
+        shallow(
+          <ModalPrompt
+            id={id}
+            title={title}
+            confirmLabel={confirmLabel}
+            cancelLabel={cancelLabel}
+            message={message}
+          />
+        );
+      }).not.toThrow();
+    });
+  });
+
+  describe('<ModalPrompt />', () => {
+    const defaultConfirmLabel: string = 'Confirm';
+    const defaultCancelLabel: string = 'Cancel';
+    let modalPrompt: ReactWrapper<IModalPromptProps, any>;
+    let modalPromptInstance: ModalPrompt;
+
+    beforeEach(() => {
+      modalPrompt = mount(
+        <ModalPrompt
+          id={id}
+          title={title}
+          message={message}
+        />,
+        { attachTo: document.getElementById('App') }
+      );
+      modalPromptInstance = modalPrompt.instance() as ModalPrompt;
+    });
+
+    afterEach(() => {
+      modalPrompt.unmount();
+      modalPrompt.detach();
+    });
+
+    it('should set default value for cancel label when not set', () => {
+      const cancelButton = modalPrompt.find('.js-cancel');
+
+      expect(cancelButton.text()).toBe(defaultCancelLabel);
+    });
+
+    it('should set default value for confirm label when not set', () => {
+      const confirmButton = modalPrompt.find('.js-confirm');
+
+      expect(confirmButton.text()).toBe(defaultConfirmLabel);
+    });
+  });
+
+  describe('<ModalPrompt />', () => {
+    let modalPrompt: ReactWrapper<IModalPromptProps, any>;
+    let modalPromptInstance: ModalPrompt;
+
+    beforeEach(() => {
+      modalPrompt = mount(
+        <ModalPrompt
+          id={id}
+          title={title}
+          confirmLabel={confirmLabel}
+          cancelLabel={cancelLabel}
+          message={message}
+        />,
+        { attachTo: document.getElementById('App') }
+      );
+      modalPromptInstance = modalPrompt.instance() as ModalPrompt;
+    });
+
+    afterEach(() => {
+      modalPrompt.unmount();
+      modalPrompt.detach();
+    });
+
+    it('should call prop onCancel when modalPrompt x is clicked and prop is set', () => {
+      const cancelSpy = jasmine.createSpy('onCancel');
```

Comment:
You are re-using the same spies in a couple of tests, you could instantiate it in a beforeEach instead of re-creating it in each test. I'm talking about `cancelSpy`, and `confirmSpy` here.